### ISSUE TITLE
Tooltip of the 'Date' control is not keyboard accessible in Windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#9696]
**Title:** Tooltip of the 'Date' control is not keyboard accessible in Windows.
 
**Test Environment:** 
OS: Windows 11 Version 24H2(OS Build 26120.1930)
Browser: Edge Version 130.0.2849.27
URL: https://www.ag-grid.com/examples/filter-date/date-filter/typescript/

**User Impact:**
The lack of keyboard accessibility for the tooltip of the 'Date' control in Windows can impede users who depend on keyboard navigation, including those with motor disabilities, from accessing crucial information. This limitation may result in confusion or mistakes when interacting with the 'Date' control. 

**Repro steps:**

1. Open the URL: https://www.ag-grid.com/examples/filter-date/date-filter/typescript/ in latest edge browser.
2. Press tab key to move to the 'date' control.
3. Enter the invalid date in the edit field.
4. Verify whether the Tooltip of the 'Date' control is keyboard accessible or not.

**Actual Result:**
Tooltip of the 'Date' control is not keyboard accessible in Windows. On mouse hover the tooltip is displayed.

**Refer Attachment:** 

<img width="969" alt="Image" src="https://github.com/user-attachments/assets/2b5c5f5c-30e3-49ed-a73b-9b52e5c736a3" />

https://github.com/user-attachments/assets/a5802d39-17e2-4acc-b522-63a7503c6234

**Expected Result:**
Tooltip of the 'Date' control should be keyboard accessible in Windows.

**Comment by AG-Zoheil** [comment]
Hello 

Thank you for the suggestion.

Currently there's no way to implement this behaviour with AG Grid. However, we agree this would be a valuable addition to the product and we have added a feature request for this.

We have added this requirement to our backlog and we are tracking it with the following reference: **AG-4278**

We have no timeline for this at the moment, but this can still be picked up by our team when they're making modifications to this functional area of AG Grid. Please use the functionality currently provided by AG Grid until this is delivered at a later point.

You can follow the status of this item on the product pipeline page::
https://ag-grid.com/pipeline/

The best way to track this is to sign up for AG Grid new release notifications using [the instructions here](https://ag-grid.zendesk.com/hc/en-us/articles/360002648058-How-do-I-sign-up-for-ag-Grid-release-notifications-). This way you'll know as soon as a new version is out and you can check whether this specific item was implemented on [the changelog page](https://www.ag-grid.com/ag-grid-changelog/).

Thanks again for bringing this up with us.

Kind regards,
Zoheil
